views.py

```diff
@@ -153,6 +153,8 @@ def page_detail(request, page_id, book_id=None):
         annotation['xml_uri'] = annot_xml_uri
         curr_annot = get_annotation_detail(annotation)
         context['annotations'].append(curr_annot)
+    if(len(context['annotations']) != 0):
```

Comment:
you should be able to just say: if context['annotations']: (it'll evaluate to False if empty)